charm_test.go

```diff
@@ -219,3 +225,88 @@ func cloneDir(c *gc.C, path string) string {
 	c.Assert(err, gc.IsNil)
 	return newPath
 }
+
+func (s *CharmSuite) assertVersionFile(c *gc.C, dir string, execName string, args []string) {
+	testing.PatchExecutableAsEchoArgs(c, s, execName)
+
+	// copy all the contents from 'path' to 'tmp folder dummy-charm'
+	// Using cloneDir
+	tempPath := cloneDir(c, dir)
+
+	// create an empty .execName file inside tempDir
+	lookupDir := "." + execName
```

Comment:
No need for explicit var, just do
vcsPath := filepath.Join(tempPath, "." + execName)

Some people would also even do

_, err := os.Create(filepath.Join(tempPath, "." + execName))

Reply:
Done